main.py

```diff
@@ -0,0 +1,65 @@
+#%%
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+quotes = [(650,1036),
+    (745,992),
+    (800,956),
+    (900,902),
+    (1200,816),
+    (1700,756),
+    (2200,642)]
+
+coverage = np.array([coverage for coverage,premium in quotes])
+premium = np.array([premium for coverage,premium in quotes])
+
+m,b = np.polyfit(coverage,premium,1)
+
+plt.plot(coverage,premium,'o')
+plt.plot(coverage,m*coverage+b)
+plt.show()
```

Comment:
Looks like this is a duplicate of what's below. Which one is the correct one?

Reply:
The bottom one...